jdatetime/__init__.py

```diff
@@ -249,6 +249,37 @@ def __init__(self, year, month, day, **kwargs):
             self.j_weekdays_short = self.j_weekdays_short_en
             self.j_ampm = self.j_ampm_en
 
+        self.strftime_mapping = {
```

Comment:
the mapping doesn't need to be public, maybe we can prefix with `_`?

Reply:
Yes, replaced it with `strftime_mapping`